iotdb-1.3/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb130/IoTDB.java

```diff
@@ -779,9 +779,13 @@ public String getInsertOneBatchSql(
         case INT64:
         case FLOAT:
         case DOUBLE:
+        case TIMESTAMP:
+        case DATE:
```

Comment:
do we need to add some prefix?

Reply:
Timestamp and date can be distinguished by value, so there is no need to add a prefix.

Reply:
![image](https://github.com/user-attachments/assets/95df2ae9-6966-4dc2-b344-e5ccc32f9631)